content/pelotech/drone-helm3/index.md

```diff
@@ -65,20 +65,44 @@ string_values
 values_files
 : Arguments for helm's `--values` flag.
 
+skip_kubeconfig
+: Whether to skip kubeconfig file creation.
+: If you already have a kubeconfig file (like in a temporary volume shared between steps), you can skip the creation of kubeconfig file.
+: For example:
+
+```yaml
```

Comment:
We are not adding code blocks to the parameter list.

Reply:
@tboerger Thanks. Then I removed the block. Please check. Thanks!